### DISCUSSION TITLE
Some feature ideas

### DISCUSSION THREAD

**Discussion by dburd08** [WSUCEG-7140/Craft#1] [Ideas]
What are some basic features we might want to add? It is important to note we have not looked at the code base yet so some of these might be more feasible than others

**Comment by dburd08** [comment]
Moving clouds
Deeper world, can not mine very deep currently
Survival mode features
Adding new block types 
Adding mobs (creatures to interact with in the world)

**Comment by TonePoems** [comment]
Sprinting mode for flying and walking

**Reply by TonePoems** [reply]
As a player, I want to be able to sprint as a movement option so that I can get places quicker (Does this mean I really just want a faster horse or a car....??)
As a hacker (I play in creative mode with flight) I want to be able to move faster while flying so that I can outrun the plebs on the ground.

**Comment by TonePoems** [comment]
A way to view commands in game/while playing? Not sure if this already exists, will test once built

**Reply by TonePoems** [reply]
As a noob player, I want to be able to look at the controls in game so I dont have to use the readme every time I forget.

**Comment by TonePoems** [comment]
As a player with elite gaming skills, I want the ability to control the mouse sensitivity so I can win in PVP.

(In reality the controls are way too sensitive currently. This isnt a driver issue as my mouse functions as expected everywhere else in the VM)
